lib/commonOpts.ts

```diff
@@ -51,59 +62,59 @@ const addModeOptions = (yargs, { read, write }) => {
     ),
     type: 'string',
   });
-};
+}
 
-const addTestingOptions = yargs =>
-  yargs.option('qa', {
+export function addTestingOptions(yargs: Argv): Argv {
+  return yargs.option('qa', {
     describe: i18n(`${i18nKey}.options.qa.describe`),
     type: 'boolean',
     default: false,
     hidden: true,
   });
+}
 
-const addUseEnvironmentOptions = yargs =>
-  yargs.option('use-env', {
+export function addUseEnvironmentOptions(yargs: Argv): Argv {
+  return yargs.option('use-env', {
     describe: i18n(`${i18nKey}.options.useEnv.describe`),
     type: 'boolean',
     default: false,
   });
+}
 
-const setLogLevel = (options = {}) => {
+export function setLogLevel(options: Arguments<CustomOptions>): void {
   const { debug } = options;
   if (debug) {
     setLoggerLogLevel(LOG_LEVEL.DEBUG);
   } else {
     setLoggerLogLevel(LOG_LEVEL.LOG);
   }
-};
+}
 
-/**
- * Get command name from Yargs `argv`
- * @param {object} argv
- */
-const getCommandName = argv => (argv && argv._ && argv._[0]) || '';
+export function getCommandName(argv: Arguments<CustomOptions>): string {
```

Comment:
Are you sure this type is right? My hunch is that it should take in `Argv`

Reply:
Going to resolve for now but feel free to reopen.